kerl

```diff
@@ -1264,33 +1277,33 @@ _do_build() {
 
     if [ -n "$KERL_RELEASE_TARGET" ]; then
         for r_type in $KERL_RELEASE_TARGET; do
-            case $r_type in
```

Comment:
Changes in this block?

- go from `case` to `if` because there's common code and I don't want to rely of case fall-through
- we don't consider `opt` (because that's a repetition, but we silently move forward)
  - ℹ️ this is the biggest (significant) change in this update block
- we make all similar code one block and then act on options `build_type_prefix` and `kerl_release_target_subfolder` which should make it easier to understand where the differences are